### DISCUSSION TITLE
Setup Github Discussions

### DISCUSSION THREAD

**Discussion by jtfairbank** [distributeaid/distributeaid.org#569] [General]
Looks like Github has a [discussions](https://github.com/distributeaid/distributeaid.org/discussions/landing) message board feature where we can answer questions about the codebase and contributing to Distribute Aid. Given the increased interest in contributing to the landing site project, that sounds like a useful way to engage with new contributors in a structured manner.

This will require a maintainer to setup.  We likely want to add different categories / maybe seed some initial threads so it's immediately useful.

**Comment by hola-soy-milk** [comment]
Heavily in favour of this! I can set it up right away or get it going tomorrow on stream

**Comment by jtfairbank** [comment]
@hola-soy-milk let's do it tomorrow on the stream?  I think we'll be spending most of the time highlighting pathways for new contributors, and this might be an interesting thing to show them / we can get the audience engaged helping us pick the categories / initial topics, etc.

**Comment by coderbyheart** [comment]
I've just enabled Discussions, not much to set on this end, however it makes sense to create an issue template and mention it there.

Example: https://github.com/distributeaid/needs-assessment/blob/b59de0cd8187b3f5c0bebfd7c7cf4656c1acf3c4/.github/ISSUE_TEMPLATE/config.yml

**Comment by coderbyheart** [comment]
I do recommend to consider moving other issues to Discussions, so the "Issue" tracker is for bugs and concrete tasks, and "Ideas" like [this](https://github.com/distributeaid/distributeaid.org/issues/349) one without a concrete solution description are separate.

**Comment by jtfairbank** [comment]
Thanks @coderbyheart!

**Comment by Snach13** [comment]
can someone tell me how to setup this project I want to contribute after installing dependencies I don't know what kind of API tokens they r talking about any project leads here ?
